Python/ceval.c

```diff
@@ -5059,8 +5059,10 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             /* Tell C compilers not to hold the opcode variable in the loop.
                next_instr points the current instruction without TARGET(). */
             opcode = _Py_OPCODE(*next_instr);
+#ifndef NDEBUG
```

Comment:
I don't see the point of polluting stderr if the caller is able to catch SystemError. I suggest removing the printf, but please complete the SystemError error message to add the info which are currently available in the printf (opcode, line number, etc.). Maybe including the code name / function name?